src/sentry/utils/samples.py

```diff
@@ -301,10 +301,14 @@ def create_sample_event(
             data["contexts"]["trace"][key] = kwargs.pop(key)
 
     data.update(kwargs)
-    return create_sample_event_basic(data, project.id, raw=raw)
+    return create_sample_event_basic(data, project.id, raw=raw, tagged=True)
```

Comment:
@Neo-Zhixing I realize we use the same code path for load mocks as well which probably shouldn't be tagged this way

Reply:
I'll add a new argument to `create_sample_event` as well and pass this down to `create_sample_event_basic`